### PR TITLE
Rules

### LINKED ISSUE #32: Include rules in the strategy guide
Should we have a section which includes the game rules? This would allow a new player to just read the rules and the basic strategy section without having to go elsewhere.

### PR DESCRIPTION
**Pull request by davidtwco**
Currently incomplete, attempts to fix #32.

### CODE REVIEW

**Comment by TartanLlama** [comment]
Looking good so far!

**Comment by TartanLlama** [comment]
Do you think we should merge this for now? Or would you like to do some more on it first?

**Comment by davidtwco** [comment]
I'm going to take another pass on it over the weekend. I think the content is pretty alright but I might reorder it a little and clarify some parts.

**Comment by TartanLlama** [comment]
Cool, sounds good!

**Comment by davidtwco** [comment]
Above issues should be fixed and it should be rebased on top of master.

File: index.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ No. of Fascists | 1 + H | 1 + H | 2 + H | 2 + H | 3 + H | 3 + H
 
 Hitler does not know who the other Fascists are but the other Fascists all know who the other Fascists are. There is a deck of Liberal and Fascist policies. There are eleven Fascist policies and six Liberal policies in the deck. An exception to this is that in a five player game, Hitler knows who the other Fascists are.
 
+In order to reveal party membership, at the start of the game, each player closes their eyes and extends their hand. Hitler will then raise his thumb and the other Fascists will open their eyes. Once all Fascists have made eye contact and know who they are working with, Hitler proceeds to lower his thumb and the other Fascists close their eyes. At this point, everyone opens their eyes and the game begins.
```
Comment (TartanLlama) [inline index.md]: Probably best to say that one player instructs the others on what to do.

**Comment by TartanLlama** [comment]
Looks great! Just that last issue to fix and I'll push. I'll play around with the CSS to make the table look a bit better.

**Comment by davidtwco** [comment]
All good?

**Pull request merged**